distribution/docs/src/main/resources/_contents/_sources/catalog-providers-contents.adoc

```diff
@@ -1,8 +1,91 @@
+{branding} uses Solr for data storage. Whenever the {branding} Solr ingests data, it creates an index for it in the data directory.
+This data indexing enables the data to be searchable by the {branding} Solr.
 
-=== Included Catalog Providers
+The data in the data directory can then be persisted by performing a `commit` on the {branding} Solr.
+When the {branding} Solr is restarted, it reloads the data to get back the index.
 
-The following catalog providers are included with a standard installation of ${branding}:
+Catalog Providers are used to interface with Solr. The following catalog providers are included with a standard installation of ${branding}:
 
-<<_catalog_solr_embedded_provider,Solr Embedded Catalog Provider>>:: An implementation of the `CatalogProvider` interface using http://lucene.apache.org/solr/[Apache Solr] as a data store.
+<<Solr Embedded Catalog Provider,Solr Embedded Catalog Provider>>:: An implementation of the `CatalogProvider` interface using http://lucene.apache.org/solr/[Apache Solr] as a data store.
```

Comment:
Should this be `Solr Embedded Provider`? In some of the previous files `Solr Catalog Provider` was changed to `Solr Provider` and I wasn't sure if this was inconsistent. There are a couple other places where this is the case when providing details on the individual implementations, and wasn't sure if this was on purpose or not